src/main/kotlin/nl/avisi/kotlinwebtest/jdbc/Validation.kt

```diff
@@ -0,0 +1,91 @@
+/**
+ * For licensing, see LICENSE.txt
+ * @author Martijn Heuvelink
+ */
+package nl.avisi.kotlinwebtest.jdbc
+
+import nl.avisi.kotlinwebtest.ExecutionContext
+import nl.avisi.kotlinwebtest.Validator
+import nl.avisi.kotlinwebtest.ValidatorResult
+import nl.avisi.kotlinwebtest.expressions.Expression
+import nl.avisi.kotlinwebtest.expressions.ExpressionEvaluator
+import nl.avisi.kotlinwebtest.interpolateExpressions
+import java.text.SimpleDateFormat
+
+val noResultsError: String = "There are no result found with the given query"
+val notEnoughResultError: String = "There are not enough result with given index"
+
+
+class QueryValidator(private val columnName: String, private val position: Int?, private var expectedValue: Expression? = null, val regex: Boolean) : Validator<JdbcStepRequest, JdbcStepResponse> {
+
+    override fun validate(executionContext: ExecutionContext, request: JdbcStepRequest, response: JdbcStepResponse): ValidatorResult {
+        if (response.resultSet!!.isEmpty()) return failure(noResultsError)
```

Comment:
!! niet gebruiken, je resultSet is optional, dus of zorgen dat die niet optional is of null checken. !! kan een nullpointer veroorzaken